packages/video-intelligence/src/index.js

```diff
@@ -14,41 +14,59 @@
  * limitations under the License.
  */
 
+/*!
+ * @module video-intelligence
+ */
+
 'use strict';
 
-var gapic = {
-  v1beta1: require('./v1beta1/index.js'),
-};
+var v1beta1 = require('./v1beta1/index.js');
 
 const VERSION = require('../package.json').version;
 
-
 /**
- * Create an imageAnnotatorClientfor the appropriate version.
- *
- * @param {Object=} opts - The optional parameters.
- * @param {String=} opts.servicePath
- *   The domain name of the API remote host.
- * @param {number=} opts.port
- *   The port on which to connect to the remote host.
- * @param {grpc.ClientCredentials=} opts.sslCreds
```

Comment:
I removed the custom type references (`grpc.ClientCredentials` and `gax.constructSettings`) for now, because they wouldn't link to anything or make much sense. We can find a better solution when we get GAX docs out... or I can revert them if you'd still like them there.